packages/quantic/force-app/main/default/lwc/quanticSmartSnippetSource/__tests__/quanticSmartSnippetSource.test.js

```diff
@@ -1,3 +1,8 @@
+import {
+  // @ts-ignore
+  getNavigateCalledWith, // @ts-ignore
+  getGenerateUrlCalledWith,
+} from 'lightning/navigation';
```

Comment:
```suggestion
import {
  // @ts-ignore
  getNavigateCalledWith, 
  // @ts-ignore
  getGenerateUrlCalledWith,
} from 'lightning/navigation';
```

Reply:
It would be fine to mark whatever rule were ignoring here at the top of the test files too.
Or we can create a global/pattern ignore in the tsconfig to ignore this rule in anything under `quantic/**/__tests__` for example.

Reply:
I modified the `navigationmixin.d.ts` file  to include the definition of these function so no need to ignore anything.